src/logdb.cpp

```diff
@@ -0,0 +1,614 @@
+// Copyright (c) 2012 The Bitcoin developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file license.txt or http://www.opensource.org/licenses/mit-license.php.
+
+#include <stdio.h>
+#include <unistd.h>
+#include <sys/stat.h>
+
+#include "logdb.h"
+
+#define LOGDB_MAX_KEY_SIZE 0x1000
+#define LOGDB_MAX_VALUE_SIZE 0x100000
+
+static const unsigned char logdb_frameheader_magic[4] = {0xB1,0xA0,0xEE,0xC9};
```

Comment:
@Diapolo: it's a file format magic. Check: http://en.wikipedia.org/wiki/Magic_number_(programming)